app/services/profit_sharing_service.py

```
@@ -344,12 +344,13 @@ async def process_profit_sharing_payment(
                 
                 # Record transaction
                 transaction = CreditTransaction(
-                    user_id=user_id,
+                    account_id=credit_account.id,
                     amount=credits_earned,
-                    transaction_type="profit_sharing_payment",
+                    transaction_type=CreditTransactionType.BONUS,
```

Ruff (0.12.2)

Undefined name `CreditTransactionType`

(F821)